app/controllers/application_controller.rb

```diff
@@ -38,6 +38,7 @@ def set_planning_application
     application = get_planning_application(param)
 
     @planning_application = PlanningApplicationPresenter.new(view_context, application)
+    @show_header_bar = true
```

Comment:
If we're enabling this in the base controller (and changing the variable used for it) then we should also remove it from the [tasks controller](https://github.com/unboxed/bops/blob/main/app/controllers/planning_applications/assessment/tasks_controller.rb#L11) where it was initially enabled.